proposal.rst

```diff
@@ -162,7 +179,7 @@ FILLME
 .. these are few, it is usually possible to get enough done across Summer of
 .. Code to make for a worthwhile project.
 
-FILLME
+Monday–Saturday 5am-4pm UTC 
```

Comment:
I'd suggest that 11 hours a day six days a week is not really sustainable for the entire length of the programme. While this is obviously up to you, please take seriously our guidance notes in the comment above — because if you burn out in the first few weeks by pushing yourself too hard, that will almost certainly result in you being failed at one of the later checkpoints (because you'll be unable to contribute effectively after you've burned out). You can generally get as much or more done sustainably (so over an indefinite period) by keeping your regular hours down to more like 40 hours a week. While we don't want to discourage your enthusiasm, we see this as a valuable part of the programme: learning how to contribute in a sustainable fashion, whether to open source or to other kinds of jobs.